src/lib/pages/schedule-view.svelte

```diff
@@ -36,13 +40,15 @@
     scheduleId: decodeURIForSvelte(scheduleId),
   };
 
-  let scheduleFetch = fetchSchedule(parameters, fetch);
+  let scheduleFetch = fetchSchedule(parameters);
 
   let showPauseConfirmation = false;
   let showDeleteConfirmation = false;
-
   let reason = '';
 
+  let coreUser = coreUserStore();
+  let editDisabled = $coreUser.namespaceWriteDisabled(namespace);
+
   const handleDelete = async () => {
     try {
       $loading = true;
```

Comment:
**Note:** On delete, there's a flash of the schedule view with the modal open before it goes back to the schedules page.

Removing the `$loading = false;` in the `setTimeout` here should fix it.

Reply:
Yeah but then if you go to another schedule it shows 'Deleting Schedule...'. Need to set $loading = false somewhere. I think moving it to afterNavigate() will help with the flash. I want goto() to instantly redirect me somewhere but clearly thats not the case.